### PR TITLE
Fix nav+pic overlapping/ pic wrapping / add footer / correct comments

### PR DESCRIPTION
**Pull request by JelenadlS**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -195,4 +202,21 @@ a[href^="tel"]::before {
...
+.table-left{
+  text-align: left;
+  vertical-align: text-top;
+  font-size: 90%;
+}
+
+.table-right{
+  text-align: right;
+  vertical-align: text-top;
+  font-size: 90%;
```
Comment (Quaquaro) [inline styles.css:217]: Maybe try to add a width of 25% to the two table classes.

File: styles.css [hunk 2]
```diff
@@ -110,11 +114,13 @@ img {
...
 .no-bullets {
 list-style-type: none; 
+margin: 4px 10px;
 }
 
 .buttontext {
-  background-color: rgba(176, 146, 201, 0.7);
+  background-color: rgba(164, 130, 192, 0.7);
   padding: 1px 10px;
+  margin: 5px;
```
Comment (Quaquaro) [inline styles.css:123]: You could add a box-shadow to your buttons.

File: styles.css [hunk 3]
```diff
@@ -68,15 +71,16 @@ border: 1px solid none;
...
 
 #coverimg {
   border-radius: 30px 190px 80px 190px;
   box-shadow: 0px 0px 30px 1px;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-  width: auto;
+  margin: auto;
+  width: 100%;
```
Comment (Quaquaro) [inline styles.css:82]: I really like this special border radius. :-)

File: styles.css [hunk 4]
```diff
@@ -110,11 +114,13 @@ img {
 
 .no-bullets {
 list-style-type: none; 
+margin: 4px 10px;
 }
 
 .buttontext {
-  background-color: rgba(176, 146, 201, 0.7);
+  background-color: rgba(164, 130, 192, 0.7);
```
Comment (Quaquaro) [inline styles.css]: May you could choose another color, which has a higher constrast. Check this out [Color Picker](https://imagecolorpicker.com/color-code/2596be)

File: styles.css (hunk 2 above)
Comment (JelenadlS) [reply styles.css:123]: Thank you, good advise to make it more pretty :-)

File: styles.css (hunk 4 above)
Comment (JelenadlS) [reply styles.css]: haha i know, will be my task now - I chose these colors at the very beginning and didnt check the contrast calculater till now 🙈

**Pull request merged**